lib/iris/analysis/maths.py

```diff
@@ -613,14 +619,14 @@ def _broadcast_cube_coord_data(cube, other, operation_noun, dim=None):
             except iris.exceptions.CoordinateNotFoundError:
                 raise ValueError("Could not determine dimension for %s. "
                                  "Use %s(cube, coord, dim=dim)"
-                                 % (operation_noun, operation_noun))
+                                 % (operation_name, operation_name))
 
     if other.ndim != 1:
         raise iris.exceptions.CoordinateMultiDimError(other)
 
     if other.has_bounds():
-        warnings.warn('%s by a bounded coordinate not well defined, ignoring '
-                      'bounds.' % operation_noun)
+        warnings.warn('Using {!r} with a bounded coordinate is not well '
+                      'defined: ignoring bounds.'.format(operation_name))
```

Comment:
Pedantry alert: I think we should use a semi-colon, rather than a comma or a colon here.

```
, + : = ;
```
???